ansible_collections/arista/avd/roles/eos_designs/doc/ptp.md

```diff
@@ -158,15 +159,45 @@ nodes:
 #### Setting PTP Clock Identity automatically
 
 By default PTP clock identity is generated and set automatically.
+The clock identity will consist of the following:
 
-```yml
-auto_clock_identity = < true | false | default -> true >
-clock_identity = < (clock_identity_prefix = 00:1C:73 (default)) + (PTP priority 1 as HEX) + ":00:" + (PTP priority 2 as HEX) >
-```
+- a prefix which is "00:1C:73" by default
+- PTP priority 1 as HEX
+- ":00:"
+- PTP priority 2 as HEX
+
+This means, that for a node with PTP priority 1 = 30 and PTP priority 2 = 2, the PTP clock identity will automatically be set to: `00:1C:73:1e:00:01`
+
+If you prefer to have PTP clock identity be the system MAC-address of the switch, which is the default EOS behaviour, simply set auto_clock_identity: false on one or more of the levels shown below:
+
+- on the fabric level, for example FABRIC.yml
+
+  ```yml
+  ptp:
+    auto_clock_identity: < true | false | default -> true >
+  ```
+
+- per node_group, for example for all spine nodes in SPINES.yml
+
+  ```yml
+  spine:
+    defaults:
+      ptp:
+        auto_clock_identity: < true | false | default -> true >
+  ```
+
+- per node for a specific node/device inside a node_group, for example for a specific leaf in LEAFS.yml
+
+  ```yml
+  nodes:
```

Comment:
please provide the full context here.
Like
```yaml
l3leaf:
  node_groups:
    leaf1:
      nodes:
        leaf1a:
          ptp:
            auto_clock_identity: < true | false | default -> true >
```

Reply:
Fixed